app/models/user_counter.rb

```diff
@@ -0,0 +1,11 @@
+class UserCounter < ApplicationRecord
+  belongs_to :user
+
+  serialize :data, HashSerializer
+  store_accessor :data, :comments_these_7_days, :comments_prior_7_days
```

Comment:
This creates the accessor that allows us to do:

`user.counters.comments_these_7_days`

in addition to

`user.counters.data[:comments_these_7_days]`